ctapipe/utils/datasets.py

```diff
@@ -25,14 +25,28 @@
 
 logger = logging.getLogger(__name__)
 
-__all__ = ["get_dataset_path", "find_in_path", "find_all_matching_datasets"]
+__all__ = [
+    "get_dataset_path", "find_in_path", "find_all_matching_datasets",
+    "get_default_url",
+]
 
 
 DEFAULT_URL = "http://cccta-dataserver.in2p3.fr/data/ctapipe-test-data/v1.1.0/"
 
 
-def get_searchpath_dirs(searchpath=None, url=DEFAULT_URL):
+def get_default_url():
+    """Get the default download url for datasets
+
+    First tries to look-up CTAPIPE_DOWNLOAD_URL and then falls back to `DEFAULT_URL`
+    """
+    return os.getenv("CTAPIPE_DOWNLOAD_URL", DEFAULT_URL)
```

Comment:
Maybe call it CTAPIPE_DATASET_URL so that it's clear it is linked to the dataset:// input url pattern

Reply:
done